Kaggle/Titanic/titanic_analysis3.ipynb

```diff
@@ -34,11 +34,11 @@
    "execution_count": 1,
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.    for i in range(len(catColumns)):
</pre><p>Is this for loop correct?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ronald-leung/datascience_machinelearning/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>